### ISSUE TITLE
Remember per-server nick

### ISSUE THREAD

**Issue by mwisnicki** [flackr/circ#212]
At the moment last nick is used on all servers after restart

**Comment by halindrome** [comment]
Yeah if it would remember a per-server nick that would be really helpful.  I have different nicks in different environments.

**Comment by fastcat** [comment]
This is also important if using a bouncer (e.g. https://www.znc.in/) where the nick is part of the login authentication
